CMakeLists.txt

```diff
@@ -24,6 +24,10 @@ if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX OR
     ${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
   set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_RELEASE} -O3 -DNDEBUG")
+  include_directories(/usr/include/python)
```

Comment:
Can you verify that this is the correct directory? Usually on Ubuntu ```python``` refers to Python 2.xx and ```python3``` is used for Python 3.xx.